### DISCUSSION TITLE
Questions about Progess Bar?

### DISCUSSION THREAD

**Discussion by TheWicklowWolf** [israel-dryer/ttkbootstrap#453] [Q&A] [answered]
1. Is there anyway to change/add a border to the Progress Bar?
2. Can you change the size(width) of the moving part?
3. Is there a way to change the speed of the moving part?

TIA

**Comment by lupon1** [comment, marked as answer]
I imagine you are referring to an indeterminate progress bar.

1. You can place the progress bar inside a label and configure the options 'highlightthickness' and 'highlightbackground' to add an outer border to it.
2. It would be nice to be able to change it. Unfortunately, I have not found a way to do so.
3. Yes, you just need to give it as a parameter when calling the 'start' method of the progress bar. By default, it is set to 10.

**Comment by TheWicklowWolf** [comment]
@lupon1 thanks!
1. Was hoping for a dynamic option, that matches the chosen theme. 
2. Same
3. Perfect
